### PR TITLE
Ports: Add port nettle version 3.8

### PR DESCRIPTION
**Pull request by plexx-dev**

### CODE REVIEW

File: Ports/nettle/package.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port='nettle'
+version='3.8'
+useconfigure=true
```
Comment (timschumi) [inline Ports/nettle/package.sh:5]:
```suggestion
useconfigure='true'
```

File: Ports/nettle/package.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+files="https://ftp.gnu.org/gnu/nettle/nettle-${version}.tar.gz nettle-${version}.tar.gz 
+https://ftp.gnu.org/gnu/nettle/nettle-${version}.tar.gz.sig nettle-${version}.tar.gz.sig"
+auth_type=sig
+auth_import_key='343C2FF0FBEE5EC2EDBEF399F3599FF828C67298'
+auth_opts=("nettle-${version}.tar.gz.sig")
```
Comment (timschumi) [inline Ports/nettle/package.sh:10]: Please validate files using `sha256` instead of `sig`.

**Review by timschumi — CHANGES_REQUESTED** [review]

File: Ports/nettle/package.sh (hunk 2 above)
Comment (plexx-dev) [reply Ports/nettle/package.sh]: fixed

**Review by linusg — CHANGES_REQUESTED** [review]
Please quash the fixup commits

**Comment by plexx-dev** [comment]
made some really strange mistake while quashing

**Pull request closed without merging**